melodic/distribution.yaml

```diff
@@ -1219,6 +1219,21 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2018.3.25-0
     status: maintained
+  lusb:
+    doc:
+      type: hg
+      url: https://bitbucket.org/dataspeedinc/lusb
+      version: default
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/DataspeedInc-release/lusb-release.git
+      version: 1.0.9-0
+    source:
+      type: hg
```

Comment:
On the other distributions, CI is disabled using https://github.com/ros/rosdistro/pull/18197/files#diff-9ab4d5354700907c09ab7ff68efb0e9bR4957. Is it intentional to enable CI only for melodic?